pages/deposits_and_withdrawals/cli_commands.md

```diff
@@ -8,10 +8,9 @@ dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipien
 
 Example:
 ```bash
-NATIVE_TOKEN_DENOM=adv4tnt
```

Comment:
Changes in this file should be resolved now by [this PR](https://github.com/dydxprotocol/v4-documentation/pull/114). Could you revert these changes?

I agree that we can make the changes in pages/governance/proposing_a_new_market.md

Reply:
hey, @ttl33 
PR #114 still uses full path to docs.dydx.exchange - is this on purpose?

Reply:
Now resolved, only the changes in `propose_a_new_market` should be reviewed